barretenberg/cpp/src/barretenberg/vm/avm/trace/gadgets/merkle_tree.cpp

```diff
@@ -0,0 +1,103 @@
+#include "barretenberg/vm/avm/trace/gadgets/merkle_tree.hpp"
+
+namespace bb::avm_trace {
+
+AvmMerkleTreeTraceBuilder::MerkleEntry AvmMerkleTreeTraceBuilder::compute_root_from_path(uint32_t clk,
+                                                                                         const FF leaf_value,
+                                                                                         const uint32_t leaf_index,
+                                                                                         const std::vector<FF>& path)
+{
+    uint32_t path_length = static_cast<uint32_t>(path.size());
+    FF curr_value = leaf_value;
+    uint32_t curr_index = leaf_index;
+    std::vector<bool> path_bits;
```

Comment:
path_bits are not assigned in this routine

Reply:
Good catch - i did not need it anymore